cmd/snapctl/commands.go

```diff
@@ -117,6 +117,21 @@ var (
 						flRunning,
 					},
 				},
+				{
+					Name: "config",
```

Comment:
That's fine as it has plugin in the front of the command and is consistent with others. Also this command only shows the global config. Do I understand this correctly?